common/metrics/defs.go

```diff
@@ -1274,6 +1278,7 @@ var ScopeDefs = map[ServiceIdx]map[int]scopeDefinition{
 		MatchingClientDescribeTaskQueueScope:                  {operation: "MatchingClientDescribeTaskQueue", tags: map[string]string{ServiceRoleTagName: MatchingRoleTagValue}},
 		MatchingClientListTaskQueuePartitionsScope:            {operation: "MatchingClientListTaskQueuePartitions", tags: map[string]string{ServiceRoleTagName: MatchingRoleTagValue}},
 		FrontendClientDeprecateNamespaceScope:                 {operation: "FrontendClientDeprecateNamespace", tags: map[string]string{ServiceRoleTagName: FrontendRoleTagValue}},
+		FrontendClientDeleteNamespaceScope:                    {operation: "FrontendClientDeleteNamespace", tags: map[string]string{ServiceRoleTagName: FrontendRoleTagValue}},
```

Comment:
@feedmeapples is migrating namespace APIs to admin side, so should this API be also moved to admin side?